README.markdown

```diff
@@ -15,7 +15,7 @@ Checkout the official Handlebars docs site at
 Installing
 ----------
 
-See our [installation documentation](http://handlebarsjs.com/installation.html).
+See our [installation documentation](https://handlebarsjs.com/guide/#installation).
```

Comment:
I think it should link to "https://handlebarsjs.com/installation/".

Reply:
I agree, the /guide/installation is just the quick-fix method for having something up and running, proper installation docs are indeed located at /installation/